### PR TITLE
Fix XCode 14 circular reference issue

### PR DESCRIPTION
**Pull request by amir-cg**

### CODE REVIEW

**Comment by rogerioth** [comment]
@v-grigoriev hello, would you be able to review and approve this pull request please? 🥺

File: SCNRecorder.podspec [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   s.name                  = 'SCNRecorder'
-  s.version               = '2.8.0'
+  s.version               = '2.8.1'
   s.summary               = 'A lags-free recorder of ARKit and SceneKit for iOS in Swift'
   s.homepage              = 'https://github.com/gorastudio/SCNRecorder'
   s.license               = { :type => 'MIT', :file => 'LICENSE.md' }
   s.author                = { 'Vladislav Grigoryev' => 'dev.grigoriev@gmail.com' }
   s.platform              = :ios, '12.0'
-  s.source                = { :git => 'https://github.com/gorastudio-ceo/SCNRecorder.git', :tag => s.version.to_s }
+  s.source                = { :git => 'https://github.com/amir-cg/SCNRecorder.git', :tag => s.version.to_s }
```
Comment (v-grigoriev) [inline SCNRecorder.podspec:9]: You have to revert this, otherwise pod will use your repo :)

Comment (amir-cg) [reply SCNRecorder.podspec]: done

**Comment by amir-cg** [comment]
@v-grigoriev which other comments?

**Comment by v-grigoriev** [comment]
> @v-grigoriev which other comments?

oh, sorry, I started review instead of adding new comments... added them now.

File: Sources/MediaSession/MediaSessionInput.swift [hunk 2]
```diff
@@ -27,30 +27,39 @@ import Foundation
...
 
-  typealias Audio = AudioMediaSessionInput
+public typealias MediaSessionInput_SampleBufferAudio = AudioMediaSessionInput & SampleBufferInput
+public typealias MediaSessionInput_SampleBufferVideo = VideoMediaSessionInput & SampleBufferInput
+public typealias MediaSessionInput_PixelBufferVideo = VideoMediaSessionInput & BufferInput
 
-  typealias Video = VideoMediaSessionInput
 
-  typealias SampleBufferAudio = AudioMediaSessionInput & SampleBufferInput
+public
```
[the remark this answers is not in the document]
Comment (amir-cg) [reply Sources/MediaSession/MediaSessionInput.swift:36]: changed into 'internal'

File: Sources/MediaSession/MediaSessionInput.swift [hunk 3]
```diff
@@ -27,30 +27,39 @@ import Foundation
...
+protocol MediaSessionInput {
 
-  typealias SampleBufferVideo = VideoMediaSessionInput & SampleBufferInput
+//  typealias Audio = AudioMediaSessionInput
```
[the remark this answers is not in the document]
Comment (amir-cg) [reply Sources/MediaSession/MediaSessionInput.swift:39]: removed

**Comment by amir-cg** [comment]
@v-grigoriev done. please review.

**Review by v-grigoriev — APPROVED** [review]

**Pull request merged**